quest/database/database.py

```diff
@@ -46,6 +46,8 @@ class Dataset(db.Entity):
         message = orm.Optional(str)
         file_path = orm.Optional(str, nullable=True)
         visualization_path = orm.Optional(str)
+        intake_plugin = orm.Optional(str)
+        intake_args = orm.Optional(str)
```

Comment:
A JSON representation of arguments, correct? If these are stored as strings, would it make sense to use the same YAML spec used by Intake text-file catalogs?